src/components/RadioGroup/__snapshots__/RadioGroup.spec.jsx.snap

```diff
@@ -292,7 +292,7 @@ exports[`RadioGroup [common] example testing should match snapshot(s) for nested
     isDisabled={false}
     isSelected={false}
     key="1"
-    name="lucid-RadioGroup-2"
+    name="8984211157799808-lucid-RadioGroup-2"
```

Comment:
Are these going to change whenever we re-run snapshots? Not a big deal, just curious if we should mock the function for the future.